commcare_connect/opportunity/models.py

```diff
@@ -697,3 +697,10 @@ class CatchmentArea(models.Model):
 
     class Meta:
         unique_together = ("site_code", "opportunity")
+
+
+class ExchangeRate(models.Model):
```

Comment:
it would be nice to have a `code, rate_date` uniqueness constraint so we only ahve one rate per currency per date